universal-login-sdk/package.json

```diff
@@ -13,7 +13,7 @@
     "fbemitter": "^2.1.1",
     "lodash.clonedeep": "^4.5.0",
     "openzeppelin-solidity": "^2.3.0",
-    "reactive-properties": "^0.1.0"
+    "reactive-properties": "^0.1.3"
```

Comment:
Remember to first commit/push, then publish. Now current version in [repo](https://github.com/UniversalLogin/universal-login-website) is 0.1.2